Game/Pregnancy/MenstrualCycle.gd

```diff
@@ -276,12 +311,11 @@ func getRoughChanceOfBecomingPregnant() -> float:
 	roughChance = clamp(roughChance, 0.02, 0.95)
 	return roughChance * 100.0
 
-func onEggCellReadyForBirth(_egg):
-	#print("EGG READY TO BIRTH")
-	emit_signal("readyToGiveBirth")
-	if(!noticedReadyToGiveBirth):
-		noticedReadyToGiveBirth = true
-		emit_signal("readyToGiveBirthOnce")
+#func onAllFetusReadyForBirth():
+#	emit_signal("readyToGiveBirth") #NOT USED AFTER https://github.com/Alexofp/BDCC/commit/49602bb2e9056017593462cb35af3daabadce38b
```

Comment:
well if its dead code its better to remove it fully, no need to leave the commented code